diesel/src/pg/backend.rs

```diff
@@ -138,18 +138,18 @@ impl SqlDialect for Pg {
     type SelectStatementSyntax = sql_dialect::select_statement_syntax::AnsiSqlSelectStatement;
 
     type ExistsSyntax = sql_dialect::exists_syntax::AnsiSqlExistsSyntax;
-    type ArrayComparison = PgStyleArrayComparision;
+    type ArrayComparison = PgStyleArrayComparison;
 }
 
 impl DieselReserveSpecialization for Pg {}
 impl TrustedBackend for Pg {}
 
 #[derive(Debug, Copy, Clone)]
-pub struct PgOnConflictClaues;
+pub struct PgOnConflictClause;
 
-impl sql_dialect::on_conflict_clause::SupportsOnConflictClause for PgOnConflictClaues {}
-impl sql_dialect::on_conflict_clause::SupportsOnConflictClauseWhere for PgOnConflictClaues {}
-impl sql_dialect::on_conflict_clause::PgLikeOnConflictClause for PgOnConflictClaues {}
+impl sql_dialect::on_conflict_clause::SupportsOnConflictClause for PgOnConflictClause {}
+impl sql_dialect::on_conflict_clause::SupportsOnConflictClauseWhere for PgOnConflictClause {}
+impl sql_dialect::on_conflict_clause::PgLikeOnConflictClause for PgOnConflictClause {}
 
 #[derive(Debug, Copy, Clone)]
-pub struct PgStyleArrayComparision;
+pub struct PgStyleArrayComparison;
```

Comment:
I'm not sure if it's a public API change, if so, we need to inform people about it :)

Reply:
That type is private as well.